lib/omb-prompt-base.sh

```diff
@@ -212,7 +212,7 @@ function git_prompt_minimal_info {
     # Output the git prompt
     SCM_PREFIX=${SCM_THEME_PROMPT_PREFIX}
     SCM_SUFFIX=${SCM_THEME_PROMPT_SUFFIX}
-    echo -e "${SCM_PREFIX}${SCM_BRANCH}${SCM_STATE}${SCM_SUFFIX}"
+    _omb_util_print "${SCM_PREFIX}${SCM_BRANCH}${SCM_STATE}${SCM_SUFFIX}"
```

Comment:
All the similar changes have the same subtlety. One needs to check the impact on all the calls of these functions.